src/Aspire.Dashboard/Components/Pages/ConsoleLogs.razor.cs

```diff
@@ -168,7 +168,7 @@ private async ValueTask LoadLogsAsync()
             else
             {
                 _initialisedSuccessfully = false;
-                _status = Loc[_selectedResource is ContainerViewModel
+                _status = Loc[_selectedResource.ResourceType is "Container"
```

Comment:
is there a reason you're not using `KnownResourceTypes.Container` here?

Reply:
Changed to use the `IsContainer()` extension method (and one other instance in this file).